service/service_construction.go

```diff
@@ -272,23 +283,42 @@ func (s ConstructionService) ConstructionParse(
 	if !ok {
 		return nil, wrapError(errInvalidInput, fmt.Errorf("%s is not a valid address", tx.To))
 	}
+	var opMethod string
+	var value *big.Int
+	// Erc20 transfer
+	if len(tx.Data) != 0 {
+		if !mapper.EqualFoldContains(s.config.TokenWhiteList, tx.To) {
```

Comment:
This may be a bit of a dangerous check in case they modify this list with inflight txs. I think it is ok to not do this check in offline mode.